src/jit/gentree.h

```diff
@@ -2334,10 +2334,85 @@ struct GenTreeColon: public GenTreeOp
         {}
 };
 
-/* gtCall   -- method call      (GT_CALL) */
+// gtCall   -- method call      (GT_CALL)
 typedef class fgArgInfo *  fgArgInfoPtr;
 enum class InlineObservation;
 
+// Multi-reg return type descriptor of a GT_CALL node.
+// x64 Unix, Arm64, Arm32 and x86 allow a value to be returned in multiple
```

Comment:
I don't consider this a short-term path.  For singl-reg scalar returns gtType on call node fully describes return value.  Replacing gtType with gtReturnDesc for call nodes would be a much bigger change.